go-controller/hybrid-overlay/pkg/controller/master.go

```diff
@@ -216,7 +221,7 @@ func (m *MasterController) handleOverlayPort(node *kapi.Node, annotator kube.Ann
 
 	// we need to setup a reroute policy for hybrid overlay subnet
 	// this is so hybrid pod -> service -> hybrid endpoint will reroute to the DR IP
-	if len(config.HybridOverlay.ClusterSubnets) > 0 {
+	if len(config.HybridOverlay.ClusterSubnets) >= 0 {
```

Comment:
is there any point in even having this check? Isn't this always true?

Reply:
Before this patch, we ask users always to configure `ClusterSubnets` for the hybrid overlay nodes, and ovnkube will assign a subnet for each HO node. This patch allows a hybrid overlay configuration without explicitly configuring `ClusterSubnets`, which is the `=0`  case. Under such a condition, the HO node subnet shall already be assigned by 3rd party component (e.g. the CNO in the migration scenario).

Reply:
fixed